src/BloomExe/Book/BookCopyrightAndLicense.cs

```diff
@@ -303,19 +313,50 @@ public static void SetOriginalCopyrightAndLicense(HtmlDom dom, BookData bookData
 			{
 				return; //leave the original there.
 			}
-			var metadata = BookCopyrightAndLicense.GetMetadata(dom);
+			var encodedCopyrightAndLicense = GetOriginalCopyrightAndLicenseNotice(collectionSettings, GetMetadata(dom));
+			bookData.Set("originalCopyrightAndLicense", encodedCopyrightAndLicense, "*");
+			bookData.Set("originalLicenseUrl", GetLicenseUrl(dom), "*");
+			bookData.Set("originalCopyright", System.Web.HttpUtility.HtmlEncode(BookCopyrightAndLicense.GetMetadata(dom).CopyrightNotice), "*");
+			bookData.Set("originalLicenseNotes", dom.GetBookSetting("licenseNotes").GetFirstAlternative(), "*");
+			bookData.RemoveAllForms("copyright");  // RemoveAllForms does modify the dom
+		}
+
+		/// <summary>
+		/// This is used each time we update xmatter. It assumes originalLicenseUrl, originalCopyright, and originalLicenseNotes have
+		/// already been set while creating the book. It updates the actual displayed originalCopyrightAndLicense to the appropriate
+		/// derived value based on the current language settings of the book. It should be called before we update the DOM from
+		/// the datadiv.
+		/// </summary>
+		/// <param name="dom"></param>
+		/// <param name="bookData"></param>
+		/// <param name="collectionSettings"></param>
+		public static void UpdateOriginalCopyrightAndLicense(HtmlDom dom, BookData bookData, CollectionSettings collectionSettings)
```

Comment:
I think this should be redone as a method parallel to BookData.SetUpDisplayOfTopicInBook(). It should not write anything to the data-div. Instead, like SetUpDisplayOfTopicInBook(), it should search the book for @data-derived='original....notice' (whatever the exact name is).  

In this way, we make it obvious both in code and in the book itself that these are not new pieces of "data"; they are projections of the original metadata which PR now saves, onto the page.